src/crafting_gui.cpp

```diff
@@ -878,7 +878,8 @@ static bool query_is_yes( const std::string &query )
     const std::string subquery = query.substr( 2 );
 
     return ( ( subquery == "yes" ) || ( subquery == "y" ) || ( subquery == "1" ) ||
-             ( subquery == "true" ) || ( subquery == "t" ) || ( subquery == "on" ) );
+             ( subquery == "true" ) || ( subquery == "t" ) || ( subquery == "on" ) ||
+             ( subquery == _( "yes" ) ) );
```

Comment:
You can remove nearly all the parentheses in this statement, only the ones around `"yes"` are needed, the others are redundant.

Reply:
Yes, I thought about that.  I guess maybe it's worth doing while I'm here.

Reply:
Thanks, for helping out so quickly! Is the "no" also covered with this?

Reply:
There's no explicit check for 'no'.  Anything that doesn't match one of the 'yes' options is treated as 'no'.

Reply:
I've updated the OP to clarify that.